doc/source/conf.py

```diff
@@ -212,7 +212,7 @@
 if 'dev' in version:
     html_theme_options["switcher"]["version_match"] = "development"
 
-if 'versionwarning' in tags:  # noqa
+if 'versionwarning' in tags:  # noqa: F821
```

Comment:
I assume this is just going over my head but is `tags` actually defined here?

Reply:
https://www.sphinx-doc.org/en/master/usage/configuration.html